spec/models/miq_report_spec.rb

```diff
@@ -491,6 +491,23 @@ def invoke_ae
       end
     end
 
+    context "chargeback report" do
+      let(:report) do
+        MiqReport.new(
+            :title   => "chargeback_report",
+            :db      => "Chargeback",
+            :cols    => %w(start_date display_range entity_namecpu_cost cpu_metric cpu_used_cost
+                           cpu_used_metric fixed_compute_1_cost fixed_compute_2_cost fixed_cost
+                           memory_cost memory_metric memory_used_cost memory_used_metric total_cost),
+            :include => {})
+      end
+      let(:ems) { FactoryGirl.create(:ems_vmware, :zone => @server.zone) }
+
+      it "runs report" do
+        report.generate_table(:userid => "admin")
```

Comment:
> The memoized helpers are never triggered in this spec. If they should already exist, perhaps you meant to do let!?

This was shamelessly copied from the spec above it so im not sure about the ```let```




> You have no expectation here; I assume this code causes an exception? Prefer expect { report.generate_table(:userid => "admin") }.not_to raise_error for better RSpec feedback, etc.

```report.generate_table(:userid => "admin") }.not_to raise_error``` should be the way to go.


anything about the ```NotImplementedError```? :/